README.md

```diff
@@ -16,6 +16,7 @@ The following parameters are required by this module
 - `alert_desc` A description of the alert.
 - `app_insights_query` The custom query to execute against the logs. See https://docs.microsoft.com/en-us/azure/application-insights/app-insights-analytics .
 - `action_group_name` The name of the action group to invoke when the alert is triggered.
+- `custom_email_subject` The subject of the email sent to the email IDs specified in the action group. (If there are no email IDs in the action group, this must still be defined but can be set to the empty string.)
```

Comment:
> this must still be defined but can be set to the empty string.
does that actually work?
if it does work then set the variables default to be an empty string and don't mention the above sentence, either way i would not mention the sentence as we only support email alerts atm

Reply:
We only support email alerts from the action-group module, but we could use this to make alerts to pre-existing or manually-defined action groups, which might not have any email recipients. If we leave a blank default then if there are email recipients and the consumer doesn't think they have to define it, there might be a blank email subject...

That was my thought process anyway. Please confirm

Reply:
it'll do for now